### DISCUSSION TITLE
在无边框的前提下使窗口最大化不遮挡任务栏？

### DISCUSSION THREAD

**Discussion by Saboteur18** [hooke007/mpv_PlayKit#292] [mpv 通用]
如题，想要相对沉浸的视频体验同时不错过下方任务栏的消息，但是无边框下最大化会遮挡任务栏；如果使用边框则可以快捷地最大化，但边框的白色实在太明显，同时我没有找到办法把mpv的边框颜色单独调为黑色，请问有什么解决办法吗？
![mpv](https://user-images.githubusercontent.com/42571927/228327161-cfe5e631-bff8-4d9c-ba65-f09e05b4b6f5.png)
这是mpv边框化的显示
![potplayer](https://user-images.githubusercontent.com/42571927/228327173-4d70fd97-8a39-4e11-90a3-d7f58e81b4f4.png)
potplayer在使用窗口化图层输出时的显示（边框在鼠标移至顶端时出现，不了解原理，具体设置如下）
![potplayer settings](https://user-images.githubusercontent.com/42571927/228329736-a087b692-4139-41ac-8faf-e7fbb1bd1be2.png)

目前我使用预设比例为2560×1400的窗口大小输出，但是一旦改变就无法快捷调回，求解

**Comment by hooke007** [comment]
无边框模式失去了作为窗口的特性。
建议等待上游的pr合并
https://github.com/mpv-player/mpv/pull/11469
https://github.com/mpv-player/mpv/pull/11473

> potplayer在使用窗口化图层输出时的显示

pot自己绘制的边框，mpv用的是系统的，无法比较。

> 目前我使用预设比例为2560×1400的窗口大小输出，但是一旦改变就无法快捷调回

如果你是指 --geometry 在windows平台目前无法在运行时改变

**Reply by Saboteur18** [reply]
明白了，非常感谢

**Reply by hooke007** [reply]
win11已解决
`border=no` 不应被推荐使用
`title=no` 作为替代方案
